### PR TITLE
Fix/71994 patch jquery resize

### PR DESCRIPTION
**Pull request by niconerd**
https://central.tri.be/issues/84807

Not sure about the use of TBD inside the js. Will that be replaced by the packager or should I hardcode the version number?

### CODE REVIEW

**Comment by borkweb** [comment]
I'd go ahead and hardcode the version number there. We don't analyze the JS files in our `mt tbd` script (though, we should probably add that at some point)

File: vendor/jquery-resize/jquery.ba-resize.js [hunk 1]
```diff
@@ -191,6 +191,17 @@
...
           data = $.data( this, str_data );
+
+        /**
+         * The Events Calendar Patch
+         *
+         * @since TBD
+         * @see https://github.com/cowboy/jquery-resize/issues/1
+         * @see https://central.tri.be/issues/71994
+         */
+        if (!data) {
```
Comment (borkweb) [inline vendor/jquery-resize/jquery.ba-resize.js]: Change spacing on this line to: `if ( ! data ) {`

File: vendor/jquery-resize/jquery.ba-resize.js [hunk 2]
```diff
@@ -191,6 +191,17 @@
...
+          data = $.data( this, str_data, {})
```
Comment (borkweb) [inline vendor/jquery-resize/jquery.ba-resize.js]: Add a space after the `}` and add a semicolon at the end of the line

**Comment by niconerd** [comment]
thanks for the review @borkweb and for the help on this one!

**Pull request merged**